app/upgrades/v10/constants.go

```diff
@@ -0,0 +1,8 @@
+package v10
+
+const (
+	// UpgradeName is the shared upgrade plan name for mainnet
+	UpgradeName = "v10.0.0"
```

Comment:
We haven't discussed how we will deal with release candidates here. 

We can add this to `main` but then adjust the handler for each release on the release branch `v10.0.x`. Once we create a final final release, the change would have to be merged to `main` again. What do you think @fedekunze @facs95 ?

Reply:
same for the binaries